### PR TITLE
DO-1771 installation and upgrade doc update.

### PR DESCRIPTION
**Pull request by Kaali09**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/server-installation/core-services.md [hunk 1]
```diff
@@ -76,7 +78,10 @@ Update **core_vault_sunbird_api_auth_token**, **core_vault_kong__test_jwt** and
 | KeycloakRealm | Creates Sunbird Realm. After the Sunbird realm is created, configure Keycloak by using the steps mentioned in the **Keycloak Configuration** section. |
 | Learner | Deploys the Learner Service. Handles user management and helps to search content. |
 | Lms | Deploys the Lms Service. It provides the APIs for lms functionality of Sunbird.|
-| Content | Deploys the content service. Helps to create content. |  
+| Content | Deploys the content service. Helps to create content. | 
+| certTemplate | Uploads static certification template to storage |
+| EncService | Deploys the encryption service. Helps to generate the encryption keys |
+| Cert | Deploys the cert service. Helps to issue the certificates |
```
Comment (Sunitashelar) [inline developer-docs/server-installation/core-services.md]: @Kaali09 should it be 'Deploys the certificate service' or is 'cert service' the name of the service?

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/upgrade/build_n_deploy.md [hunk 2]
```diff
@@ -22,57 +22,45 @@ As part of this upgrade, you may choose to set up the load balancer for swarm ma
...
 
-4. After plugins build and deploy, provide the blob url of the plugins zip file in Player build. Refer Player build job on [this page](developer-docs/server-installation/artifactupload-job/core-services){:target="_blank"}
+4. After the plugins build and deploy, provide the blob url of the plugins zip file in the Player build. For details, refer to the Player build job on the [Core Services](developer-docs/server-installation/artifactupload-job/core-services){:target="_blank"} page.
 
-5. Build all the services mentioned in the table below. In case of Cassandra migration (Core / Cassandra), the job needs to be built twice and deployed thrice. Refer to the tags page to get the two tags. Also in case of errors in Cassadnra migration, refer to [this page](developer-docs/server-installation/core-services){:target="_blank"}
+5. Build all the services mentioned in the table below. In case of Cassandra migration (Core / Cassandra), the job needs to be built twice and deployed thrice. Refer to the tags page to get the two tags. In case of errors in Cassadnra migration, refer to the [Core Services](developer-docs/server-installation/core-services){:target="_blank"} page.
```
Comment (Sunitashelar) [inline developer-docs/upgrade/build_n_deploy.md]: @Kaali09 could you please provide an exact link to the tags page?

File: developer-docs/upgrade/build_n_deploy.md [hunk 3]
```diff
@@ -22,57 +22,45 @@ As part of this upgrade, you may choose to set up the load balancer for swarm ma
...
+|                         | 	                  | Yarn               |	Yarn (Multiselect all options in the job parameter job_names_to_deploy)	             | EncService           | EncService |
+|  	                  |                       |                    | AnalyticsGeoLocationDBSetup | Cert           | Cert |
 
 
-**Note:** 
-1. We have disabled Git LFS for private repositories. Hence, uninstall git lfs from your private repository.
-To do so, clone your private repository and run ```git lfs uninstall``` and also remove the ```.gitattributes``` file
+> **Note:** 
+1. Refer to the following notes to trigger the **Neo4jElasticSearchSyncTool** job.
+**Steps to migrate DialcodeRequired:**
```
Comment (Sunitashelar) [inline developer-docs/upgrade/build_n_deploy.md]: @Kaali09 could you please check if the rephrasing of this sentence works? If not, please tell me what exactly do you mean, then we can phrase it differently.

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/upgrade/build_n_deploy.md (hunk 3 above)
Comment (Kaali09) [reply developer-docs/upgrade/build_n_deploy.md]: @Sunitashelar I have Worked on your comment. Please verify.

File: developer-docs/upgrade/build_n_deploy.md (hunk 2 above)
Comment (Kaali09) [reply developer-docs/upgrade/build_n_deploy.md]: @Sunitashelar I have worked on your comment. pleaese review.

File: developer-docs/server-installation/core-services.md (hunk 1 above)
Comment (Kaali09) [reply developer-docs/server-installation/core-services.md]: @Sunitashelar I have worked on your comment. please review.

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/server-installation/core-services.md (hunk 1 above)
Comment (Sunitashelar) [reply developer-docs/server-installation/core-services.md]: done

File: developer-docs/upgrade/build_n_deploy.md (hunk 3 above)
Comment (Sunitashelar) [reply developer-docs/upgrade/build_n_deploy.md]: done

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Kaali09,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://541.qa.docs.sunbird.org/2.5.0/ .<br/> **Note:** Only _2.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/ansible-variables/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/core-services/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/current_release_tags_n_jenkins_jobs/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/knowledge-platform/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/server-installation/plugins/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/build_n_deploy/ <br /> - http://541.qa.docs.sunbird.org/2.5.0/developer-docs/upgrade/update_ansible_variables/ <br />  <br/> **Modified sections are -** <br />

**Pull request merged**